appBringup is ready

imirica opened the pull request:
Please review my appBringup

imirica committed:
appBringup is ready

dmgciubotaru commented:
Use this
https://docs.python.org/3/library/tkinter.html#a-simple-hello-world-program
putting it in a class will make it easier to extend later

dmgciubotaru reviewed (changes requested):
use this class instead of plain code

imirica commented:
sir, yes sir :))

imirica committed:
appBringup halfway

#condul a fost transpus intr-o clasa
#ignora designe-ul actual
#button_expore este functional
#button_exit  este functional

P.S. Ma gandesc sa folosesc variabila "filename" ca input pentru scriptul "duplicateFinder", in care introduceam path-ul pentru cautarea fisierelor duplicate cu "sys.argv[1]"
